docs/sources/tutorials/play-with-grafana-mimir/docker-compose.yml

```diff
@@ -2,8 +2,8 @@ version: '3.4'
 services:
   minio:
     image: minio/minio
+    entrypoint: [""]
```

Comment:
What is the difference between `entrypoint: ""` and `entrypoint: [""]`? I don't see this explained when looking at the [documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#entrypoint).

Reply:
It's exec form vs shell form, explained here:
https://docs.docker.com/engine/reference/builder/#entrypoint

Reply:
Thank you, good to know. It may be worth adding comment about this into the yaml file itself?

Reply:
Thanks for looking at this and adding the doc! 

After deeper diving into this, I got the following findings:
- The position of `entrypoint` is actually makes no difference, but I will leave it like this (because `entrypoint` will be executed before `cmd`) unless there are other opinions.
- It is kinda hard to put a comment there, as I believe this is a [bug](https://github.com/docker/compose/pull/8585) in Docker where `entrypoint=""` is equivalent to nothing in this case, instead of clearing the entry point. It is just a coincidence that this bug didn't happen in the `exec` form, but not directly related to the  `exec` form. However, it seems to have been fixed in the newer Docker version. 

Do you think we should add the Docker bug information as a comment to this `yaml` file?